src/Sylius/Bundle/WebBundle/Resources/config/routing/backend/content/block/action_block.yml

```diff
@@ -26,8 +26,9 @@ sylius_backend_action_block_update:
         _controller: sylius.controller.action_block:updateAction
         _sylius:
             template: SyliusWebBundle:Backend/Content/Block/Action:update.html.twig
-            method: find
-            arguments: [$id]
+            repository:
+                method: find
+                arguments: [$id]
```

Comment:
After looking into this, it not use the config component, so it is not really simple to use this notation ...
Probably use the config component for routing configuration _sylius node can be a good idea ? @pjedrzejewski @aRn0D